### PR TITLE
Move tools locally

### PR DESCRIPTION
**Pull request by Tanza3D**

### CODE REVIEW

File: tools/src/stdev_pp_calc/README.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+---
+owners-github: remryia
```
Comment (minusQuantumNeko) [inline tools/src/stdev_pp_calc/README.md:2]: Bro xD
```suggestion
owners-github: remyria
```

**Review by minusQuantumNeko — CHANGES_REQUESTED** [review]

File: tools/src/stdev_pp_calc/README.md (hunk 1 above)
Comment (Tanza3D) [reply tools/src/stdev_pp_calc/README.md:2]: remryia

**Review by minusQuantumNeko — DISMISSED** [review]
Fix CODEOWNERS, but everything else should be okay

**Review by EXtremeExploit — APPROVED** [review]
ok ig, i want input from @minusQuantumNeko to know if his tool is up to date just to be sure

**Comment by Tanza3D** [comment]
can tell u now it probably is, looks up to date to me

**Comment by minusQuantumNeko** [comment]
I think it is @EXtremeExploit

**Review by minusQuantumNeko — APPROVED** [review]

**Pull request merged**